clib/src/main/scala/scala/scalanative/libc/stdlib.scala

```diff
@@ -2,10 +2,13 @@ package scala.scalanative
 package libc
 
 import scalanative.unsafe._
+import scala.scalanative.unsigned.ULong
 
 @extern
 object stdlib {
 
+  type size_t = ULong
```

Comment:
Picky, picky: Please consider `type size_t = CSize"
That has better, to me, semantics. You can also then drop
an import.

In a better world, then the rest of the file would use `size_t` but I understand
the virtue of respecting prior art.